packages/strapi-plugin-content-manager/services/permission-checker.js

```diff
@@ -43,7 +43,10 @@ const createPermissionChecker = ({ userAbility, model }) => {
   const sanitizeCreateInput = data => sanitizeInput(ACTIONS.create, data);
   const sanitizeUpdateInput = entity => data => sanitizeInput(ACTIONS.update, data, entity);
 
-  const buildPermissionQuery = query => permissionsManager.queryFrom(query);
+  const buildPermissionQuery = (query, action) => permissionsManager.queryFrom(query, action);
+
+  const buildReadQuery = query => buildPermissionQuery(query, ACTIONS.read);
+  const buildDeleteQuery = query => buildPermissionQuery(query, ACTIONS.delete);
```

Comment:
Why declaring helpers only for `read` & `delete`? Is it to keep it simple as long as we don't need the other ones?